### PR TITLE
Add interchangeable for and in for sprint command

### PR DESCRIPTION
**Pull request by marvelman3284**
Add feature #67

### CODE REVIEW

**Comment by cwarwicker** [comment]
Hi,

I've checked this out and tried it on my local but it doesn't seem to work. Running `sprint in 5` for example, gives me 'invalid sprint command'.

Also, looking at the `opt2` variable, it looks like you could do `sprint in 5 now` which doesn't make any sense.

Can you have another look at this please?

File: cogs/writing/sprint.py [hunk 1]
```diff
@@ -103,6 +104,27 @@ async def sprint(self, context, cmd=None, opt1=None, opt2=None, opt3=None):
...
+        elif cmd == "in":
+
+            delay = opt1
+
+            # If the second option is invalid, display an error message
+            if opt2 is not None and opt2.lower() != "for":
+                return await context.send(
+                    user.get_mention()
+                    + ", "
+                    + lib.get_string("sprint:err:for:unknown", user.get_guild())
```
Comment (cwarwicker) [inline cogs/writing/sprint.py:116]: Can you add a new error message to the lang file, as this says the format should be `sprint for 20 in 5` whereas in this command it should be `sprint in 5 for 20`. So add another lang string eg. `sprint:err:in:unknown` and change the format of the error.

File: cogs/writing/sprint.py [hunk 2]
```diff
@@ -103,6 +104,27 @@ async def sprint(self, context, cmd=None, opt1=None, opt2=None, opt3=None):
...
+                )
+
+            # Get the length they want before starting the sprint.
+            # If they left off the last argument and just said `sprint for 20` then assume they mean now.
+            if opt2 is None:
+                length = 20
```
Comment (cwarwicker) [inline cogs/writing/sprint.py:122]: Can you please use the constant for this, rather than hard-coding the value `20`. Should be `self.DEFAULT_LENGTH`

**Review by cwarwicker — CHANGES_REQUESTED** [review]
Added a few comments for changes. Almost there I think.

File: cogs/writing/sprint.py (hunk 1 above)
Comment (marvelman3284) [reply cogs/writing/sprint.py]: Done.

File: cogs/writing/sprint.py (hunk 2 above)
Comment (marvelman3284) [reply cogs/writing/sprint.py]: Done.

**Review by cwarwicker — APPROVED** [review]

**Pull request merged**